.jenkins/Packaging.Jenkinsfile

```diff
@@ -14,7 +14,8 @@ def packageUpload(String version, String build_type) {
                            cmake ${WORKSPACE} -DCMAKE_BUILD_TYPE=${build_type} -DCMAKE_INSTALL_PREFIX:PATH='/opt/openenclave' -DCPACK_GENERATOR=DEB
                            make
                            ctest --output-on-failure --timeout ${CTEST_TIMEOUT_SECONDS}
-                           make package
+                           cpack
+                           cpack -D CPACK_DEB_COMPONENT_INSTALL=ON -DCPACK_COMPONENTS_ALL=OEHOSTVERIFY
                            """
                 oe.Run("clang-7", task)
                 azureUpload(storageCredentialId: 'oe_jenkins_storage_account', filesPath: 'build/*.deb', storageType: 'blobstorage', virtualPath: "master/${BUILD_NUMBER}/ubuntu/${version}/${build_type}/SGX1FLC/", containerName: 'oejenkins')
```

Comment:
Why is this package uploaded under SGX1FLC?

Reply:
Technically, the package was built with USE_LIBSGX=ON, so we upload those under SGX1FLC build directories. It's a little confusing, and the USE_LIBSGX flag will probably be changing to something like USE_DCAP in the future, so we'll rename these directories to something like SGX1DCAP, maybe? A lot of the naming is awkward right now.